hazelcast/src/main/java/com/hazelcast/dataconnection/HazelcastDataConnection.java

```diff
@@ -64,6 +64,11 @@ public class HazelcastDataConnection extends DataConnectionBase {
      */
     public static final String CLIENT_YML_PATH = "client_yml_path";
 
+    /**
+     * IMap resource type name
+     */
+    public static final String OBJECT_TYPE_IMAP = "IMap";
```

Comment:
Strictly speaking, this should be `IMapJournal` - that's what we support in Jet source, but we don't support it in SQL yet, so no real harm done.

Reply:
HazelcastDataConnection in theory supports all objects in HZ. We use it in `remoteMapJournal` but why there is no `remoteMap` with `DataConnectionRef`?

Reply:
It just didn't get into 5.3 scope.

Reply:
renamed. If we start supporting IMap we will add another object type here.
6b07e8964e5a92db48616fd460330956f936d252